portfolio/src/main/webapp/index.html

```diff
@@ -7,106 +7,132 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="getMessage();createMap()"> 
-      
-      <div class = "headerofpage"> 
-	      <h1>Kerry O'Malley</h1>
-	      <h3> Portfolio </h3>
-      </div> 
-	
-      <button class = "tabs active" onclick= "openPage(event, 'AboutMe')" id = "defaultOpen"  >About Me</button> 
-      <button class = "tabs" onclick= "openPage(event, 'Programming')">Programming</button>
-      <button class = "tabs" onclick= "openPage(event, 'FunFacts')">Fun Facts</button>
-      <button class = "tabs" onclick= "openPage(event, 'Pics')">Pictures</button> 
+  <body onload="getMessage();createMap();userLogin()"> 
+    <!-- Header Section -->
+    <div class="headerofpage"> 
+      <h1>Kerry O'Malley</h1>
+      <h3> Portfolio </h3>
+    </div> 
+    <!-- Login status and button -->
+    <div class="login-part">
+      <div id="user-login-content"></div>
+    </div>
+   
+    <button class="tabs active" onclick="openPage(event, 'AboutMe')" id="defaultOpen">About Me</button> 
+    <button class="tabs" onclick="openPage(event, 'Programming')">Programming</button>
+    <button class="tabs" onclick="openPage(event, 'FunFacts')">Fun Facts</button>
+    <button class="tabs" onclick="openPage(event, 'Pics')">Pictures</button> 
 
+    <!-- About me tab -->
+    <div id="AboutMe" class="innercontent" style=display:block> 
+      <h1 class="center">Introduction</h1>
+      <p class="center">Hi! My name is Kerry, and I am a STEP Intern 
```

Comment:
So for multi-line paragraphs, structure them like this
```
<p>
  content...
</p>
```
Do this here and below